playbook.yaml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
This really exists for the sole purpose of starting with a OpenShift 4.0 cluster and adding the stuff to test KubeVirt with OLM.